apps/client/src/features/rundown/event-editor/EventEditorEmpty.module.scss

```diff
@@ -22,6 +22,17 @@
   border-collapse: separate;
   border-spacing: 4rem 0;
 
+  kbd {
+    border-radius: 2px;
+    border: none;
+    background-color: #262626; // $gray-1200
+    padding: 0.125rem 0.5rem;
+    color: #f6f6f6; // $ui-white
+    font-weight: 400;
+    box-shadow: 0px 0px 3px 0px rgba(0, 0, 0, 0.4);
+    font-size: calc(1rem - 2px);
+  }
```

Comment:
```suggestion
  kbd {
    border-radius: 2px;
    border: none;
    background-color: $gray-1200;
    padding: 0.125rem 0.5rem;
    color: $ui-white;
    font-weight: 400;
    box-shadow: 0px 0px 3px 0px rgba(0, 0, 0, 0.4);
    font-size: calc(1rem - 2px);
  }
```

Reply:
Resolved in https://github.com/cpvalente/ontime/pull/1438/commits/3dc1fa107f3fbf4721f15de758862faebd49aea1